src/Strategy/Python/Pipenv.hs

```diff
@@ -24,13 +24,15 @@ import Data.Set (Set)
 import Data.Text (Text)
 import Data.Text qualified as Text
 import DepTypes
+import Diag.Common
```

Comment:
[optional nit]: Can we make the imports for this file explicit while we're here?